package.json

```diff
@@ -25,34 +25,34 @@
     "test:all": "jest --config=utils/test/jest.config.js"
   },
   "devDependencies": {
-    "@types/react": "16.4.13",
-    "@zendeskgarden/css-variables": "5.0.4",
+    "@types/react": "16.4.14",
+    "@zendeskgarden/css-variables": "5.1.0",
     "@zendeskgarden/eslint-config": "7.0.5",
     "@zendeskgarden/stylelint-config": "9.0.5",
     "@zendeskgarden/svg-icons": "4.4.5",
     "babel-cli": "6.26.0",
     "babel-core": "6.26.3",
     "babel-eslint": "9.0.0",
-    "babel-jest": "23.4.2",
+    "babel-jest": "23.6.0",
     "babel-loader": "7.1.5",
     "babel-plugin-add-header-comment": "1.0.3",
     "babel-plugin-inline-react-svg": "0.5.4",
-    "babel-plugin-styled-components": "1.6.3",
+    "babel-plugin-styled-components": "1.7.1",
     "babel-plugin-transform-es2015-modules-commonjs": "6.26.2",
     "babel-plugin-transform-object-assign": "6.22.0",
     "babel-preset-es2015": "6.24.1",
     "babel-preset-react": "6.24.1",
     "babel-preset-stage-0": "6.24.1",
     "chalk": "2.4.1",
-    "commander": "2.17.1",
+    "commander": "2.18.0",
     "coveralls": "3.0.2",
     "enhanced-resolve": "4.1.0",
     "enzyme": "3.6.0",
     "enzyme-adapter-react-16": "1.5.0",
     "enzyme-to-json": "3.3.4",
-    "eslint": "5.5.0",
-    "eslint-config-prettier": "3.0.1",
-    "eslint-plugin-jest": "21.22.0",
+    "eslint": "5.6.0",
```

Comment:
Ideally, we should revert this back to 5.5.0 as it is not the expected peer dependency for `@zendeskgarden/eslint-config`. However, if the rest of the PR merges clean, it's not a big deal.

Reply:
With https://github.com/zendeskgarden/eslint-config/pull/24/files being merged, are we good to allow this to be upgraded?

Reply:
Yup, it's all good now.